src/Overflow/Overflow.tsx

```diff
@@ -1,26 +1,38 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { props, t, ReactNode } from '../utils';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { props, t, ReactElement } from '../utils';
 import ResizeSensor from '../ResizeSensor/ResizeSensor';
 
+export type OverflowProps = {
+  /** react node initially rendered */
+  content: JSX.Element,
+  /** react node rendered if `content` overflows its parent */
+  contentIfOverflowing: JSX.Element,
+  id?: string,
+  className?: string,
+  style?: React.CSSProperties
+};
+
+export type State = {
+  isOverflowing: boolean
+};
+
 export const Props = {
-  content: ReactNode,
-  contentIfOverflowing: ReactNode,
+  content: ReactElement,
```

Comment:
`Overflow` never supported a content that was a `string` or a `number` so I explicitly made it stricter here and in the TS type